scss/bitstyles/settings/_global.color-palette.scss

```diff
@@ -64,17 +64,17 @@ $bitstyles-color-palette: (
   ),
   'background': (
     '100': $bitstyles-base-color-background,
-    '95': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 5%),
-    '90': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 10%),
-    '80': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 20%),
-    '70': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 30%),
-    '60': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 40%),
-    '50': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 50%),
-    '40': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 60%),
-    '30': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 70%),
-    '20': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 80%),
-    '10': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 90%),
-    '5': mix($bitstyles-base-color-white, $bitstyles-base-color-background, 95%),
+    '95': mix($bitstyles-base-color-brand-1, $bitstyles-base-color-background, 3%),
```

Comment:
Just to double check: the 3% is on purpose?

Reply:
No! Thanks, good catch 🙇‍♂️

Reply:
Aha, I checked this — yes it’s deliberate, though it certainly looks like an error. I’m not sure how to handle it better right now 🤔 I’ll add a comment.

TLDR: Because nothing related to color and color perception is easy, if I mix 5% of the brand color with white, the output is still very dark — switching to 3% makes a large difference, and is much closer to the background color, as would be expected I think